src/RageSoundPosMap.cpp

```diff
@@ -181,12 +162,8 @@ std::int64_t pos_map_queue::Search( std::int64_t iSourceFrame, bool *bApproximat
 	static RageTimer last;
 	if( last.PeekDeltaTime() >= 1.0f )
 	{
-		last.GetDeltaTime();
-		std::stringstream ss;
-		ss << "Approximate sound time: driver frame " << iSourceFrame << ", m_pImpl->m_Queue frame "
-		   << pClosestBlock->m_iDestFrame << ".." << (pClosestBlock->m_iDestFrame+pClosestBlock->m_iFrames)
-		   << " (dist " << iClosestPositionDist << "), closest position is " << iClosestPosition;
-		LOG->Trace( "%s", ss.str().c_str() );
+		last.Touch(); 
```

Comment:
If we don't use the value of last anywhere. Do we even need it?

Reply:
We don't, that's why previously I replaced it with a function which only reset the timer.

Reply:
I think we can just get rid of it then.

Reply:
Works for me!